packages/apollo-utils/src/sentry/apolloSentryPlugin.ts

```diff
@@ -52,7 +52,7 @@ export const sentryPlugin: ApolloServerPlugin<BaseContext> = {
         for (const err of ctx.errors) {
           // Only report internal server errors,
           // errors extending ApolloError should be user-facing
-          if (NO_REPORT_ERRORS.has(err.extensions?.code?.toString())) {
+          if (NO_REPORT_ERRORS.has(err.extensions?.code?.toString() ?? '')) {
```

Comment:
Rather than looking for an empty string in the set, I just wouldn't do the lookup if you don't have that key

e.g.
```
const errCode = err.extensions?.code?.toString()
if (errCode && NO_REPORT_ERRORS.has(errCode) {...}
```